client/service/src/client/wasm_override.rs

```diff
@@ -238,12 +233,8 @@ mod tests {
 			match scraped {
 				Err(e) => {
 					match e {
-						sp_blockchain::Error::Msg(msg) => {
-							let is_match = msg
-								.matches("Duplicate WASM Runtimes found")
-								.map(ToString::to_string)
-								.collect::<Vec<String>>();
-							assert!(is_match.len() >= 1)
+						sp_blockchain::Error::DuplicateWasmRuntime(duplicates) => {
+							assert_eq!(duplicates.len(), 1);
```

Comment:
this was `>= 1` before, any reason why you changed it?
```suggestion
							assert!(duplicates.len() >= 1);
```

Reply:
The impl reports one for each duplicate, there is one duplicate. So the bound here was needlessly too weak.

Reply:
If it's always one, why are you using a `Vec`? :)

Reply:
It's guaranteed for the test case to be always one, should have been more explicit.